website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,270 @@
+![Advanced Table expandable rows. The parent rows display a summary of a Hashicorp product and the total price, the children rows show a breakdown of each billing item from that product and their individual cost.](/assets/components/table/advanced-table/expandable-rows.png) 
+
+### When to use
+
+Use expandable rows to display hierarchical data efficiently. This allows users to explore details within the table without navigating away.
+
+!!! Dont
+
+Avoid using expandable rows when data is not structured in parent-child relationships.
+
+![Advanced Table where the parent row has cells for name and email, but children rows have cells containing order date and total.](/assets/components/table/advanced-table/advanced-table-dont-parent-nested.png)
+
+!!!
+
+<!-- ### Interaction
+
+- Users can navigate between cells using arrow keys. To expand or collapse a row, they must first focus on the expand button using the arrow keys, then press Enter to activate it. -->
+
+### Nested rows
+
+Use the same density setting for parent and child rows to keep layouts consistent.
+
+
+!!! Dont
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/advanced-table-density-mix.png)
+
+!!!
+
+
+### Striping 
+
+!!! Info
+
+Ensure that content within striped rows continue to maintain adequate color contrast with the striped background.
+
+!!!
+
+![Advanced Tables with row striping have rows that alternate between white and light grey background color.](/assets/components/table/advanced-table/advanced-table-striping.png)
+
+Striping enhances readability by alternating row colors, making it easier to scan tabular data.
+
+- Non-Nested Advanced Tables: Striping starts with the second row, distinguishing it from the header.
+- Nested Advanced Tables: Child rows are automatically striped, while parent rows remain unstriped to visually reinforce hierarchy. This behavior cannot be disabled.
+
+### Placement
+
+!!! Info
+
+The row placement property is only relevant within Figma and doesn’t exist as a property within the code.
+
+!!!
+
+Row placement determines the visual styling based on where the row is placed relative to other rows within the Advanced Table. Only cells with a column placement that is either start or end use the row placement property; column position middle does not use this property.
+
+![The cell with column placement end and row placement end has a border radius set on the bottom right corner.](/assets/components/table/advanced-table/table-row-placement.png)
+
+## Cells
+
+### Density
+
+- We recommend using medium cell density by default.
+- If the content is largely text-based, the short density allows more content to be displayed within the page.
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Horizontal scrolling
+
+- We recommend using this when there are many columns of equal priority.
```

Comment:
Would they really be of equal priority, though? I would expect columns to generally be organized from higher priority (left) to lower priority (right). 

Something like this is more accurate:

```suggestion
Use horizontal scrolling when the number of columns expands beyond the viewport or container.
```